extensions/blocks/map/map-theme-picker/style.scss

```diff
@@ -1,13 +1,14 @@
+@import '../../../shared/styles/gutenberg-colors.scss';
 
 .component__map-theme-picker__button {
 	.edit-post-settings-sidebar__panel-block & {
-		border: 1px solid lightgray;
+		border: 1px solid $light-gray-500;
 		border-radius: 100%;
 		width: 56px;
 		height: 56px;
 		margin: 2px;
 		text-indent: -9999px;
-		background-color: lightgray;
+		background-color: $light-gray-500;
```

Comment:
Not exactly the same but very close and listed in Gutenberg as _"good for "grayed" items and borders."_